cypress/integration/proposal-booking.spec.ts

```diff
@@ -300,9 +308,7 @@ context('Proposal booking tests ', () => {
 
         cy.get('[data-cy="add-equipment-responsible"]').click();
 
-        cy.get('input[type="checkbox"]')
```

Comment:
I have seen many changes like this, where the formatting is adjusted. Maybe we should make one commit called 'formatting adjustments'?